internal/services/azapi_client_config_data_source.go

```diff
@@ -50,6 +52,15 @@ func (r *ClientConfigDataSource) Schema(ctx context.Context, request datasource.
 			"subscription_id": schema.StringAttribute{
 				Computed: true,
 			},
+
+			"subscription_resource_id": schema.StringAttribute{
+				Computed:            true,
+				MarkdownDescription: "The resource ID of the subscription. E.g. `/subscriptions/00000000-0000-0000-0000-000000000000`",
+			},
+
+			"object_id": schema.StringAttribute{
+				Computed: true,
```

Comment:
Please add a `MarkdownDescription` for this field.

Reply:
Done :)